js/views/modals/Settings/Store.js

```diff
@@ -149,7 +150,9 @@ export default class extends baseVw {
   }
 
   getSettingsData() {
-    const selected = this.modsSelected.selectedIDs;
+    let selected = app.settings.get('storeModerators');
+    // The mods may not have loaded in the interface yet. Subtract only explicitly de-selected ones.
+    selected = _.without(selected, ...this.modsSelected.unselectedIDs);
```

Comment:
This could also be accomplished just as concisely with the native Array.filter().